app/routes/questionnaire.py

```diff
@@ -165,47 +165,40 @@ def post_questionnaire(schema, questionnaire_store):
     return redirect(router.get_first_incomplete_location_in_survey().url())
 
 
-@questionnaire_blueprint.route("sections/<section_id>/", methods=["GET"])
-@questionnaire_blueprint.route("sections/<section_id>/<list_item_id>/", methods=["GET"])
+# pylint: disable=too-many-return-statements
+@questionnaire_blueprint.route("sections/<section_id>/", methods=["GET", "POST"])
+@questionnaire_blueprint.route(
+    "sections/<section_id>/<list_item_id>/", methods=["GET", "POST"]
+)
```

Comment:
I think it may have got lost in my last review comments, but we should consider making this `GET` only and using a link styled as a button on the section summary page. This would mean one less `POST` request on every section summary in completion of a questionnaire. As this wasn't captured in the initial card we could break this out into a separate piece of work.

Reply:
Got to go separate story, I am starting to get frustrated and need to get this checked in